client/my-sites/plans/main.jsx

```diff
@@ -259,6 +261,7 @@ class Plans extends Component {
 		}
 
 		const hideFreePlan = ! is2023PricingGridVisible || this.props.isDomainAndPlanPackageFlow;
+		const planTypes = this.props.jetpackAppPlans ? [ TYPE_PERSONAL, TYPE_PREMIUM ] : null;
```

Comment:
`Non-blocking / nitpick`:

Maybe consider adding a small inline comment here?

// The jetpack app uses a query param to only display two plans -- personal and premium